pinot-common/src/main/java/org/apache/pinot/common/config/GrpcConfig.java

```diff
@@ -28,6 +28,24 @@ public class GrpcConfig {
   public static final String GRPC_TLS_PREFIX = "tls";
   public static final String CONFIG_USE_PLAIN_TEXT = "usePlainText";
   public static final String CONFIG_MAX_INBOUND_MESSAGE_BYTES_SIZE = "maxInboundMessageSizeBytes";
+
+  private static final String CONFIG_CHANNEL_SHUTDOWN_TIMEOUT_SECOND = "channelShutdownTimeoutSecond";
+  private static final int DEFAULT_CHANNEL_SHUTDOWN_TIMEOUT_SECOND = 10;
+
+  // KeepAlive configs
+  private static final String CONFIG_CHANNEL_KEEP_ALIVE_ENABLED = "channelKeepAliveEnabled";
+  // To control keep alive on no usage
+  private static final boolean DEFAULT_CHANNEL_KEEP_ALIVE_ENABLED = false;
+
+  private static final String CONFIG_CHANNEL_KEEP_ALIVE_TIME_SECONDS = "channelKeepAliveTimeSeconds";
+  private static final int DEFAULT_CHANNEL_KEEP_ALIVE_TIME_SECONDS = 300; // 5 minutes
```

Comment:
Consider merging this 2? Basically enabling keep alive is equivalent to setting the keep alive time to a positive value

Reply:
Yeah, I had thought about that, there's a default value though. And then we will have to differentiate between default vs not-set vs explicitly set (but to default). Can revisit in follow up.